eng/WpfArcadeSdk/tools/CodeGen/DesignTimeTextTemplating.targets

```diff
@@ -0,0 +1,98 @@
+      TemplatesToProcess="@(T4TransformInputs)"
+      IncludeFolders="$(IncludeFolders)"
+      DirectiveProcessors="@(DirectiveProcessor)"
+      AssemblyReferences="@(T4AssemblyReference)"
+      ReferencePaths="@(T4ReferencePath)"
+      TrackerLogDirectory="$(TrackerLogDirectory)"
+      TrackFileAccess="$(TrackFileAccess)"
+      MinimalRebuildFromTracking="$(TransformOutOfDateOnly)"
+      OverwriteReadOnlyOutputFiles="$(OverwriteReadOnlyOutputFiles)"
+      ParameterValues="@(T4ParameterValues)"
+      >
+
+      <!-- List of output files that were generated by the task.
+      If a read-only version of the file was overwritten by the task, the
+      task item for that file will have the following metadata entry:
+        <ReadOnlyFileOverwritten>true<ReadOnlyFileOverwritten>
+      -->
+      <Output ItemName="GeneratedFiles" TaskParameter="GeneratedFiles"/>
+
+      <!-- List of output files that could not be generated because the file file
+      already exists and is read-only. -->
+      <Output ItemName="NonGeneratedFiles" TaskParameter="NonGeneratedFiles"/>
+    </Microsoft.VisualStudio.TextTemplating.Build.Tasks.TransformTemplates>
+
+    <PropertyGroup>
+      <!-- Unless another namespace has been specified, use the project namespace as the
+      default namespace from pre-processed files. -->
+      <PreprocessTemplateDefaultNamespace Condition=" $(PreprocessTemplateDefaultNamespace)=='' ">$(RootNamespace)</PreprocessTemplateDefaultNamespace>
+    </PropertyGroup>
+
+    <PreprocessTemplates
+      DefaultNamespace="$(PreprocessTemplateDefaultNamespace)"
+      TemplatesToProcess="@(T4PreprocessInputs)"
+      IncludeFolders="$(IncludeFolders)"
+      DirectiveProcessors="@(DirectiveProcessor)"
+      AssemblyReferences="@(T4AssemblyReference)"
+      ReferencePaths="@(T4ReferencePath)"
+      TrackerLogDirectory="$(TrackerLogDirectory)"
+      TrackFileAccess="$(TrackFileAccess)"
+      MinimalRebuildFromTracking="$(TransformOutOfDateOnly)"
+      OverwriteReadOnlyOutputFiles="$(OverwriteReadOnlyOutputFiles)"
+      ParameterValues="@(T4ParameterValues)"
+      >
+
+      <!-- List of output files that were generated by the task.
+      If a read-only version of the file was overwritten by the task, the
+      task item for that file will have the following metadata entry:
+        <ReadOnlyFileOverwritten>true<ReadOnlyFileOverwritten>
+      -->
+      <Output ItemName="GeneratedFiles" TaskParameter="GeneratedFiles"/>
+
+      <!-- List of output files that could not be generated because the file file
+      already exists and is read-only. -->
+      <Output ItemName="NonGeneratedFiles" TaskParameter="NonGeneratedFiles"/>
+
+      <!-- List of assemblies that are required to run the processed code. -->
+      <Output ItemName="T4RequiredAssemblies" TaskParameter="RequiredAssemblies"/>
+    </PreprocessTemplates>
+  </Target>
+  <!-- End workaround -->
+  <ItemGroup>
+    <T4ParameterValues Include="NewtonsoftLocation">
```

Comment:
Are you just trying to get to the full path of `Newtonsoft.Json.dll` within the Nuget Package? 

If so, you will likely want to add a dependency to [`NewtonSoft.Json`](https://github.com/dotnet/arcade/blob/f42c6d1de75067f8c610f160006ce02187da683d/Documentation/ArcadeConventions.md), then set [`GeneratePathProperty=true`](https://github.com/NuGet/Home/issues/7851) for the corresponding `PackageReference` (which you could likely add right here). 

Nuget would generate an MSBuild property for you like `$(PkgNewtonSoft_Json)`, which will point you to the root of the package on the file-system.